pylint/reporters/json_reporter.py

```diff
@@ -75,7 +75,7 @@ class JSONReporter(BaseJSONReporter):
     TODO: 3.0: Remove this JSONReporter in favor of the new one handling abs-path
     and confidence.
 
-    TODO: 2.16: Add a new JSONReporter handling abs-path, confidence and scores.
+    TODO: 3.0: Add a new JSONReporter handling abs-path, confidence and scores.
```

Comment:
Here I think we should just upgrade the TODO, we can't release a new 2.X version easily, let's just do it in 3.0.